Extension/README.md

```diff
@@ -53,6 +53,7 @@ The extension has platform-specific binary dependencies, therefore installation
 :--- | :---
 `cpptools-linux.vsix` | Linux 64-bit
 `cpptools-linux-armhf.vsix` | Linux ARM
+`cpptools-linux-aarch64.vsix` | Linux ARM 64-bit
```

Comment:
I went with using aarch64 instead of arm64 since that's what the architecture is reported as on the 64-bit Raspberry Pi OS (same reason I used armhf instead of just arm)...let me know if you if you think I should change that to arm64.